x-pack/metricbeat/module/aws/aws_test.go

```diff
@@ -48,38 +47,3 @@ func TestGetRegions(t *testing.T) {
 	assert.Equal(t, 1, len(regionsList))
 	assert.Equal(t, regionName, regionsList[0])
 }
-
-func TestConvertPeriodToDuration(t *testing.T) {
-	cases := []struct {
-		period               time.Duration
-		expectedDuration     string
-		expectedPeriodNumber int
-	}{
-		{
-			period:               time.Duration(300) * time.Second,
-			expectedDuration:     "-10m0s",
-			expectedPeriodNumber: 300,
-		},
-		{
-			period:               time.Duration(10) * time.Minute,
-			expectedDuration:     "-20m0s",
-			expectedPeriodNumber: 600,
-		},
-		{
-			period:               time.Duration(30) * time.Second,
-			expectedDuration:     "-1m0s",
-			expectedPeriodNumber: 30,
-		},
-		{
-			period:               time.Duration(60) * time.Second,
-			expectedDuration:     "-2m0s",
-			expectedPeriodNumber: 60,
-		},
-	}
-
-	for _, c := range cases {
-		duration, periodSec := convertPeriodToDuration(c.period)
-		assert.Equal(t, c.expectedDuration, duration)
-		assert.Equal(t, c.expectedPeriodNumber, periodSec)
-	}
-}
```

Comment:
[The best code is no code at all](https://blog.codinghorror.com/the-best-code-is-no-code-at-all/) :slightly_smiling_face: